roles/gitlab/tasks/gitlab.yml

```
@@ -44,6 +46,8 @@
   when:
   - "'uuid' in gitlab_jail and (gitlab_jail['uuid'] is defined and gitlab_jail['uuid'] != '-')"
   iocage: state=pkg tag={{ production_jails.gitlab.tag }} cmd="install -y {{ production_jails.gitlab.pkgs | join(' ') }}"
+  ignore_errors: yes
```

As above - please don't ignore errors on things we care about.